src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -189,22 +190,21 @@ public abstract class GraphQLHttpMiddleware
     public GraphQLHttpMiddleware(
         RequestDelegate next,
         IGraphQLTextSerializer serializer,
-        GraphQLHttpMiddlewareOptions options /*,
-        IEnumerable<IWebSocketHandler>? webSocketHandlers = null */)
+        GraphQLHttpMiddlewareOptions options,
+        IEnumerable<IWebSocketHandler>? webSocketHandlers = null)
     {
         _next = next ?? throw new ArgumentNullException(nameof(next));
         _serializer = serializer ?? throw new ArgumentNullException(nameof(serializer));
         Options = options ?? throw new ArgumentNullException(nameof(options));
-        //_webSocketHandlers = webSocketHandlers;
-        //_userContextBuilderForWebSockets = new UserContextBuilder<IDictionary<string, object?>>(BuildUserContextAsync);
+        _webSocketHandlers = webSocketHandlers;
+        _userContextBuilderForWebSockets = new UserContextBuilder<IDictionary<string, object?>>(BuildUserContextAsync);
     }
 
     /// <inheritdoc/>
     public virtual async Task InvokeAsync(HttpContext context)
     {
         if (context.WebSockets.IsWebSocketRequest)
         {
-            /************* WebSocket support ************
```

Comment:
I'd remove `// Process WebSocket request `comment  below since `HandleWebSocketAsync` is pretty obvios and also has xml comments.